docker/dev/.env

```diff
@@ -5,5 +5,6 @@ CLIENT_ID='xx'
 CLIENT_SECRET='xx'
 OAUTH_SECRET='random-secret-string'
 ENV_JS_FILEPATH='/home/Desktop/DTaaS/deploy/config/client/env.local.js'
+OAUTH_CONF_FILEPATH='/home/Desktop/DTaaS/docker/conf.auth'
```

Comment:
This variable is good to have so that developers can check their changes with and without the backend authorization.